R/repl.R

```diff
@@ -7,8 +7,10 @@
 #' 
 #' @import utils
 #' 
+#' @param banner Boolean; print a startup banner when launching the REPL?
+#' 
 #' @export
-py_repl <- function() {
+python_repl <- function(banner = TRUE) {
```

Comment:
maybe `quiet = FALSE` (to match the R `--quiet` startup option and the python `-q` startup option)

Reply:
I am thinking that the IDE may indeed want to show this banner by default as it documents which version of Python is being used.

Reply:
We may want a `module` argument that allows the user to specify that they want to interact with a specific module (e.g. "keras" or "tensorflow"). We can then call `import(module)` to force reticulate to bind to a version of Python that has this module.

Reply:
Should we allow for a vector of modules?